src/mapstreeview.cpp

```diff
@@ -60,30 +64,30 @@ void MapsTreeView::_showContextMenu(const QPoint& point) {
     m_mapMenu->exec(viewport()->mapToGlobal(point));
 }
 
-void MapsTreeView::_disableActions() {
-    removeAction(m_newMapAction);
-    QObject::disconnect(m_newMapAction, SIGNAL(triggered()), this,
-                        SLOT(_onNewMapAction()));
-    QObject::disconnect(m_propertiesAction, SIGNAL(trigerred()), this,
-                        SLOT(_onPropertiesAction()));
+void MapsTreeView::disableActions() {
     delete m_newMapAction;
+    m_newMapAction = nullptr;
```

Comment:
After having destroyed the pointed object, ```pointer == nullptr``` will still return false. The pointer still points a memory adress, but the unaffected one. 

When destroying a pointer that may be re-used later (and checked !), it's better to set it to nullptr